packages/react/src/autocomplete/root/AutocompleteRoot.tsx

```diff
@@ -6,19 +6,17 @@ import { useCoreFilter } from '../../combobox/root/utils/useFilter';
 import { stringifyAsLabel } from '../../utils/resolveValueLabel';
 import type { Group } from '../../utils/resolveValueLabel';
 
-const DEFAULT_FILTER_OPTIONS = { sensitivity: 'base' } as const;
-
```

Comment:
Removed this as  it's redundant since the defaults internally use this for Combobox.